applications/PfemSolidMechanicsApplication/custom_strategies/schemes/residual_based_bossak_scheme.hpp

```diff
@@ -591,7 +591,7 @@ class ResidualBasedBossakScheme: public Scheme<TSparseSpace,TDenseSpace>
         Element::Pointer rCurrentElement,
         LocalSystemVectorType& RHS_Contribution,
         Element::EquationIdVectorType& EquationId,
-        ProcessInfo& CurrentProcessInfo) override
+        const ProcessInfo& CurrentProcessInfo) //override
```

Comment:
```suggestion
        const ProcessInfo& CurrentProcessInfo)
```

this is very dangerous and can lead to broken workflows

I strongly recommend to update the signature. In general, one should never remove `override`!

Reply:
Oh ok perfect